widget/embedded/src/containers/ExpandedQuotes/ExpandedQuotes.styles.ts

```diff
@@ -0,0 +1,52 @@
+import { darkTheme, styled } from '@rango-dev/ui';
+
+export const Container = styled('div', {
+  position: 'relative',
+  display: 'flex',
+  flexDirection: 'column',
+  borderRadius: '$primary',
+  boxShadow: '15px 15px 15px 0px rgba(0, 0, 0, 0.05)',
+  overflow: 'hidden',
+  $$color: '$colors$neutral100',
+  transition: 'width 200ms ease-in-out',
+  width: 0,
+
+  [`.${darkTheme} &`]: {
+    $$color: '$colors$neutral300',
+  },
+  backgroundColor: '$$color',
+  height: '700px',
+
+  variants: {
+    show: {
+      true: {
+        width: 390,
+      },
+    },
+  },
+});
```

Comment:
I believe there are duplications in the styles, specifically in relation to the layout styles. Could you consider reusing the layout styles to eliminate redundancy? This way, if any styles are modified, we won't have to update them in two separate locations.

Reply:
Yes, but if I use layout, the animation will not be very interesting

Reply:
Maybe we can consider the same classNames for both to eliminate redundancy.

Reply:
Is it fixed?